citus-enterprise.spec

```diff
@@ -1,20 +1,19 @@
-%global pgmajorversion 11
-%global pgpackageversion 11
+%global pgmajorversion 10
+%global pgpackageversion 10
```

Comment:
Not really necessary as these values should be updated by `citus_dev` eventually.